src/userStorage.ts

```diff
@@ -1,19 +1,26 @@
-import { first } from 'rxjs';
 import { IGetUser } from './users/interface/get-user-interface';
 import { IPostUser } from './users/interface/post-user-interface';
 import { IPatchUser } from './users/interface/patch-user-interface';
 import { IDeleteUser } from './users/interface/delete-user-interface';
+import * as bcrypt from 'bcrypt';
+import { saltOrRounds } from './constans';
+import { ICreateUser } from './users/interface/create-user-interface';
+import { ISingUpUser } from './users/interface/singup-user.-interface';
+
+type Exp = number;
 
 interface UsersTable {
   [key: number]: IGetUser;
+  passwd: { [key: number]: string };
+  logout: { [key: number]: Exp };
 }
```

Comment:
better to not to have all types and interfaces in one file, please move them to directories)
And you can use logout: { [key: number]: number };